pennylane/qinfo/transforms.py

```diff
@@ -306,10 +306,11 @@ def _make_probs(tape, wires=None, post_processing_fn=None):
     if wires is None:
         wires = tape.wires
 
-    with qml.tape.QuantumTape() as new_tape:
+    with qml.queuing.AnnotatedQueue() as q:
         for op in tape.operations:
             qml.apply(op)
         qml.probs(wires=wires)
+    qscript = qml.tape.QuantumScript.from_queue(q)
```

Comment:
```suggestion
    qscript = qml.tape.QuantumScript(tape.operations, [qml.probs(wires=wires)])
```
But again, not that big of a deal.

Reply:
will do, why not